package.json

```diff
@@ -26,7 +26,7 @@
     "webpack-dev-server": "^4.15.1"
   },
   "dependencies": {
-    "@splitsoftware/browser-rum-agent": "0.3.1",
+    "@splitsoftware/browser-suite": "^1.0.1-rc.0",
```

Comment:
Change to the stable version `1.0.0` if possible, although `^1.0.1-rc.0` is fine anyway.